src/tool_help.c

```diff
@@ -863,16 +864,64 @@ static const struct feat feats[] = {
   {"alt-svc",        CURL_VERSION_ALTSVC},
 };
 
-void tool_help(void)
+static void print_category(curlhelp_t category)
+{
+  int i;
+  for(i = 0; helptext[i].opt; ++i)
+    if(helptext[i].categories & category) {
+      printf(" %-19s %s\n", helptext[i].opt, helptext[i].desc);
+    }
+}
+
+/* Prints category if found. If not, it returns 1 */
+static int get_category_content(const char *category)
+{
+  int i;
+  for(i = 0; categories[i].opt; ++i)
+    if(strcmp(categories[i].opt, category) == 0) {
+      printf("%s: %s\n", categories[i].opt, categories[i].desc);
+      print_category(categories[i].category);
+      return 0;
+    }
+  return 1;
+}
+
+/* Prints all categories and their description */
+static void get_categories()
```

Comment:
This needs a `void` within the parentheses:
~~~
tool_help.c:891:13: error: function declaration isn’t a prototype [-Werror=strict-prototypes]
  891 | static void get_categories()
      |             ^~~~~~~~~~~~~~